packages/block-library/src/navigation/editor.scss

```diff
@@ -42,7 +42,7 @@
 	}
 
 	.wp-block-navigation .block-editor-block-list__block::before {
-		left: 0;
+		left: 1px; // fixes a slight misalignment of the left border alignment between the menu item block control and the menu item border
```

Comment:
Thanks for adding a comment with context! What caused that misalignment?

Reply:
I can't quite figure that one out. I was trying to address this via a more top-level solution (addressing the positioning of the parent container) but didn't see anything off...

Reply:
From a quick google search, I think most people who use it add it via a custom walker on the `wp_nav_menu()` php function.